### PR TITLE
test images: Adds Windows Container image support for etcd, pause

### LINKED ISSUE #77268: Kubernetes test images should include Windows images in their manifest lists
**What would you like to be added**:

/sig windows
/sig testing

/area conformance

The images used in Kubernetes tests should also include Windows images in their manifest lists. Right now, in order to run E2E tests with Windows nodes, the env var ``KUBE_TEST_REPO_LIST`` needs to be set, containing alternative repositories which contain Windows images (the ``e2eteam`` repository is mainly used). The Windows images are currently maintained and built from ``kubernetes-sigs/windows-testing``: https://github.com/kubernetes-sigs/windows-testing/blob/master/images/BuildImageManifestLists.ps1

The build script in ``kubernetes/test/images`` does not build Windows images at the moment.

**Why is this needed**:

Running E2E tests on Windows nodes will be made easier, with the official Kubernetes test images.

Secondly, Hybrid OS tests will be able to be run.

### PR DESCRIPTION
**Pull request by claudiubelu**
**What type of PR is this?**

/kind feature

/sig testing
/sig windows

Commit was separated from #77269

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Partially Fixes #77268

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by mkumatag** [comment]
@claudiubelu 
etcd and pause images are maintained in a different location under here https://github.com/kubernetes/kubernetes/tree/master/build/pause and https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd, so I'm just wondering whether we can enable there only is no concerns.

**Review by mkumatag — CHANGES_REQUESTED** [review]
@claudiubelu
etcd and pause images are maintained in a different location under here https://github.com/kubernetes/kubernetes/tree/master/build/pause and https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd, so I'm just wondering whether we can enable there only is no concerns.

**Comment by claudiubelu** [comment]
> 
> 
> @claudiubelu
> etcd and pause images are maintained in a different location under here https://github.com/kubernetes/kubernetes/tree/master/build/pause and https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd, so I'm just wondering whether we can enable there only is no concerns.

Indeed.

I've sent PRs for those here:

pause image: https://github.com/kubernetes/kubernetes/pull/91452
etcd image: https://github.com/kubernetes/kubernetes/pull/92433

**Comment by marosset** [comment]
@claudiubelu Looksl like `./hack/update-bazel.sh` needs to be run again

**Comment by mkorbi** [comment]
Hello @claudiubelu
This PR has been updated frequently, but it looks like it stuck/needs a retest, so I'd like to check what's the status. The code freeze is starting 12.11.2020 (about 3-4 weeks from now) and while there is still plenty of time, we want to ensure that each PR has a chance to be merged on time.
As the PR is tagged for 1.20, is it still planned for this release?

File: test/images/pause/BUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@io_bazel_rules_go//go:def.bzl",
+    "go_binary",
+    "go_library"
```
Comment (droslean) [inline test/images/pause/BUILD:6]:
```suggestion
    "go_library",
```

**Comment by MonzElmasry** [comment]
@claudiubelu can you add the suggested code change if this pr is planned for 1.20. we have 3 days left for code freeze

**Comment by mkorbi** [comment]
Hey @claudiubelu short reminder, code freeze will be by today

**Comment by ravisantoshgudimetla** [comment]
@claudiubelu  - You can run  `./hack/update-bazel.sh` for it to be updated. Can you please run it so that we get this merged?

**Comment by jeremyrickard** [comment]
Hello, 👋 , 1.20 release lead here! We've moved this to the 1.21 release since we have passed [code freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#code-freeze) and are nearing the end of the release. If you feel that this is otherwise required for the release, please reach out in #sig-release on the kubernetes slack!

**Comment by erismaster** [comment]
Hello! I am from the Bug Triage team! This PR has not been updated for a long time, so I'd like to check on the status. The code freeze is starting March 9th, 2021 (about 1 week from now). We want to ensure that each PR has a chance to be merged before code freeze.
As the PR is tagged for 1.21, is it still planned for this release?

**Comment by dims** [comment]
has not been touched in a while dropping milestone

/milestone clear

**Comment by mkumatag** [comment]
> > @claudiubelu
> > etcd and pause images are maintained in a different location under here https://github.com/kubernetes/kubernetes/tree/master/build/pause and https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd, so I'm just wondering whether we can enable there only is no concerns.
> 
> Indeed.
> 
> I've sent PRs for those here:
> 
> pause image: #91452
> etcd image: #92433

This is not required any more I feel because of the above 2 PRs, 

/close

**Pull request closed without merging**